DEVELOPER_GUIDE.md

```diff
@@ -0,0 +1,31 @@
+## How It Works
+
+The project itself is a derivative of [finstyle](https://github.com/fnichol/finstyle), but starts with all cops disabled. Cookstyle has a direct dependency on one specific version of RuboCop (at a time), and [patches it][patch] to load the [upstream configuration][upstream] and [custom set][config] of rule updates. When a new RuboCop release comes out, this library can rev its pinned version dependency and [re-vendor](https://github.com/chef/cookstyle/blob/master/Rakefile) the upstream configuration to determine if any breaking style or lint cops were added/dropped/reversed/etc.
```

Comment:
Are there links missing for
[patches it][patch]
[upstream configuration][upstream]
[custom set][config] 
?

Reply:
yeah I think this is crazy old legacy. Let me nuke these